src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs

```diff
@@ -129,5 +129,7 @@ public string DisplayName
         public DateTime ApprovalRequestedOn;
 
         public DateTime ApprovalDate;
+        public bool IsPackageNameApproved { get; set; }
+        public string PackageNameApprovedBy { get; set; } 
```

Comment:
Is approval date interesting for anything? Looks like we have that for api approvals.

Reply:
It will only be useful to show as tooltip or for an eventual use. I will add that field so we will have that data in case required in the future. thanks